enzyme/Enzyme/Enzyme.cpp

```diff
@@ -1427,7 +1548,7 @@ class Enzyme : public ModulePass {
                        Arch == Triple::amdgcn;
 
       auto val = GradientUtils::GetOrCreateShadowConstant(
-          Logic, TLI, TA, fn, pair.second, AtomicAdd, PostOpt);
+          Logic, TLI, TA, fn, pair.second, 1, AtomicAdd, PostOpt);
```

Comment:
should this not be width?

Reply:
This is still an open question. On the one hand a function should only be shadowed once, on the other hand fwd mode currently shadows it n times and only uses the first. Also we don't necessarily know the width at this point. We would need to add width metadata to __enzyme_virtualreverse somehow. Lets keep that for later.